themes/default/layouts/page/pulumi-up-2022.html

```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
-                style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;"
-                allowfullscreen=""
-                title="PulumiUP: Announcing Pulumi 3.0"
-            ></iframe>
+        <div class="flex flex-wrap my-8">
+            <div class="w-full lg:w-1/4 border-b lg:border-b-0 lg:border-r border-white">
+                <p class="text-3xl">Thursday, May 5th</p>
+                <p>8:00AM - 12:00PM PDT<p>
+            </div>
+            <div class="w-full lg:w-3/4 lg:pl-6">
+                <p class="text-3xl">Community Summit</p>
+                <p>
+                    Join your friends in the Pulumi community for the first ever Pulumi Community Summit.
+                    This virtual event will include an “Open Spaces'' concept where you can propose and
+                    vote on topics ahead of time. Those topics with the most interest will form the agenda
+                    for the day, with virtual break out rooms centered around those themes.
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <section id="about-pulumi" class="white-background py-16">
+        <div class="max-w-4xl mx-auto px-6">
+            <h2>About Pulumi</h2>
+
+            <p>
+                Pulumi's cloud engineering platform brings infrastructure, developer, and security teams together through a
+                unified software engineering process that tames cloud complexity and accelerates innovation. Using the
+                Pulumi platform, teams can build, deploy, and manage modern cloud applications faster and with more
+                confidence, using any language, any architecture and any cloud. Pulumi lets teams build Modern Infrastructure
+                as Code using popular programming languages (Python, JavaScript, TypeScript, Go, .NET/C#). It enables
+                deploying infrastructure and applications together through a unified delivery process. Finally, teams
+                can manage cloud applications with Policy as Code, better visibility, and controls.
+            </p>
+
+            <div class="mt-8">
+                <event-session-registration-modal
+                    modal-title="Register Now"
+                    button-text="Register"
+                    button-class="btn-primary"
+                    event-sessions="{{ .Params.registration.main | jsonify }}"
+                >
+                </event-session-registration-modal>
+            </div>
+        </div>
+    </section>
+
+    <section id="ctas" class="container mx-auto py-16">
+        <div class="flex flex-wrap items-stretch">
+            <div class="w-full lg:w-1/2 p-3">
+                <div class="h-full card white-background p-6">
+                    <h3>Connect wit Peers on GitHub</h3>
+                    <p class="mb-8">Pulumi open source code is available to all. See what we are working on and interact with the community.</p>
```

Comment:
Just a quick adjustment to make it more inclusive.
```suggestion
                    <p class="mb-8">Pulumi open source code is available to all. Explore what we are working on and interact with the community.</p>
```